ReadMe-FormVer.md

```diff
@@ -12,8 +12,87 @@ dist/kotlinc/bin/kotlinc -language-version 2.0 -Xcompiler-plugin=dist/kotlinc/li
 
 ## Viper dependency
 
-TODO: Viper may need to be available locally to build the
-project, a HOWTO would be nice to have.
+To build the plugin, two main Viper dependencies are needed: 
+[silver](https://github.com/viperproject/silicon) (the API for
+generation the intermediate representation) and 
+[silicon](https://github.com/viperproject/silicon) (the symbolic engine for
+performing the verification of Viper code).
+
+However, the `silicon` *project structure* and *building file* require some
+important modifications for allowing Gradle to resolve the dependencies required by both projects.
+
+Therefore, to install the dependencies, be sure to have installed locally:
+* The Java Development Kit
+* [Maven](https://maven.apache.org/index.html)
+* [SBT](https://www.scala-sbt.org/)
+* [Z3](https://github.com/Z3Prover/z3) v4.8.7
+
+The installing of `Z3` is not necessary enough, it is important to have the JAR dependency required by `silicon`.
```

Comment:
What do you mean by "not necessary enough"?  Is it not enough to install Z3, or is it not necessary?

Reply:
I think I should clarify it.